doc-content/enterprise-only/optimizer/quarkus-creating-proj-proc.adoc

```diff
@@ -0,0 +1,130 @@
+[id='proc-quarkus-creating-proj_{context}']
+
+
+= Creating the school timetable project
+
+The school timetable project lets you get up and running with {PLANNER} and Quarkus application using Apache Maven and the Quarkus Maven plug-in.
+
+.Procedure
+
+. In a command terminal, enter the following command to verify that Maven is using JDK 11 and that the Maven version is 3.6 or higher:
++
+[source]
+----
+mvn --version
+----
+
+. If the preceding command does not return JDK {JDKVersion}, add the path to JDK {JDKVersion} to the PATH environment variable and enter the preceding command again.
+
+. To generate the project, enter one of the following commands:
++
+NOTE: Apple macOS and Microsoft Windows are not supported production environments.
++
+* If you are using Linux or Apple macOS, enter the following command:
++
+[source,shell,subs=attributes+]
+----
+mvn io.quarkus:quarkus-maven-plugin:1.11.6.Final-redhat-00001:create \
+    -DprojectGroupId=org.acme \
+    -DprojectArtifactId=optaplanner-quickstart  \
+    -Dextensions="resteasy,resteasy-jackson,optaplanner-quarkus,optaplanner-quarkus-jackson"    \
+    -DplatformGroupId=com.redhat.quarkus \
+    -DplatformVersion=1.11.6.Final-redhat-00001 \
+    -DnoExamples\
+----
++
+This command create the following elements in the `./optaplanner-quickstart` directory:
++
+** The Maven structure
+** Example `Dockerfile` file in `src/main/docker`
+** The application configuration file
++
+.Properties used in the mvn `io.quarkus:quarkus-maven-plugin:1.11.6.Final-redhat-00001:create` command
+[cols="30%,70%", options="header"]
+|===
+h| Property
+h| Description
+
+| `projectGroupId`
+| The group ID of the project.
+
+| `projectArtifactId`
+| The artifact ID of the project.
+
+| `extensions`
+| A comma-separated list of extensions to use with this project.
+
+| `platformGroupId`
+| The Group ID of the target platform.
+
+| `platformVersion`
+| The version of the platform that you want the project to use.
+
+| `noExamples`
+| Creates a project with the project structure but without tests or classes.
+
+|===
++
+The values of the `projectGroupID` and the `projectArtifactID` properties are used to generate the project version. The default project version is `1.0.0-SNAPSHOT`. The values of the `platformGroupId` and `platformVersion` properties are used by `quarkus-universe-bom` to manage project dependencies.
+
+. After the directory structure is created, open the `optaplanner-quickstart/pom.xml` file in a text editor and examine the contents of the file to ensure it contains the following elements:
+//is this the correct path to the BOM?
++
+[source,xml,subs=attributes+]
+----
+<properties>
+	...
+	<quarkus-plugin.version>1.11.6.Final-redhat-00001</quarkus-plugin.version>
```

Comment:
Hard-coded Quarkus version. Suggestion: introducing an Asciidoc attribute (variable) for the Quarkus version can save some work with find & replace in the future when the Quarkus version changes.

Reply:
Done